charts/prometheus-pushgateway/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v1.8.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 2.12.0
+version: 2.12.1
```

With a new feature, let us raise the minor number instead.
```suggestion
version: 2.13.0
```